Python/pywarpx/picmi.py

```diff
@@ -532,6 +532,8 @@ def init(self, kw):
             self.psatd_update_with_rho = kw.pop('warpx_psatd_update_with_rho', None)
             self.psatd_do_time_averaging = kw.pop('warpx_psatd_do_time_averaging', None)
             self.psatd_use_damp_fields_in_z_guard = kw.pop('warpx_use_damp_fields_in_z_guard', None)
+            self.psatd_damp_fields_nz = kw.pop('warpx_damp_fields_nz', None)
+            self.psatd_zero_fields_nz = kw.pop('warpx_zero_fields_nz', None)
```

Comment:
Hi Dave, I started looking at the PR and I have a quick question:
What are the WarpX input parameters that correspond to the ones assigned here in the PICMI interface?

Reply:
Thanks for reviewing the PR. These are left over from the changes to the damping that I removed. I'll delete these two lines.